src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -71,6 +68,16 @@ public partial class VariableView : UserControl, ICommandTarget, IDisposable {
             _environmentProvider.RefreshEnvironmentsAsync().DoNotWait();
         }
 
+        private void OnUIThemeChanged(object sender, EventArgs e) {
+            SetImageBackground();
+        }
+
+        private void SetImageBackground() {
```

Comment:
This fragment is duplicated across 3 controls. We could define some `ThemedUserControl` and derive from it, but that would be a much more extensive change.